src/taipy/rest/api/resources/datanode.py

```diff
@@ -270,7 +324,17 @@ class DataNodeWriter(Resource):
       tags:
         - api
       summary: Write into a data node
-      description: Write into a data node
+      description: |
+        Write data from request body into a data node by datanode_id. If the data node does not exist, a 404 error is returned.
```

Comment:
'Write data from request body into a data node by datanode_id.'
->
'The data stored in the request body is written to the data node whose identifier is \*datanode_id\*.'